pennylane/math/single_dispatch.py

```diff
@@ -131,6 +131,15 @@ def _permutation_parity(perm):
 ar.register_function("scipy", "linalg.det", _det_sparse)
 ar.register_function("scipy", "linalg.eigs", sp.sparse.linalg.eigs)
 ar.register_function("scipy", "trace", lambda x: x.trace())
+ar.register_function("scipy", "reshape", lambda x, new_shape: x.reshape(new_shape))
+ar.register_function("scipy", "real", lambda x: x.real)
+ar.register_function("scipy", "imag", lambda x: x.imag)
+ar.register_function("scipy", "size", lambda x: np.prod(x.shape))
+
+# Even scipy's own sum falls back to numpy. So we simply fallback entirely here
+ar.register_function(
+    "scipy", "sum", lambda x, axis: sp.sparse.coo_array(x.toarray().sum(axis=axis))
+)

```

Comment:
Can we have a backlog a task to develop a more performant sum? Seems like we shouldn't need to create a fully dense array for this.

Reply:
For now without support for sparse states, I think we do not even need to have it for now. I'll double check, and also add a `sum` story to the epic that records all the todo's for scipy.sparse missing functionalities

Reply:
sc-85323